packages/vitest/src/node/cli/cli-config.ts

```diff
@@ -582,6 +582,10 @@ export const cliOptionsConfig: VitestCLIOptions = {
   clearScreen: {
     description: 'Clear terminal screen when re-running tests during watch mode (default: true)',
   },
+  compare: {
```

Comment:
I don't think it should be here. This config is for `vitest` command, `vitest bench` has its own commands (that inherits this config)

Reply:
We had different config options before (like `--json` for benchmark), looks like I removed some of them when CLI was rewritten 🤔 

Adding `compare` to `UserConfig` should be fine, it has all "cli-only" options

Reply:
Looking at pre config schema era https://github.com/vitest-dev/vitest/pull/5126/files#diff-c322bc45ad914ffa25dc80a6918be23a154d9a5e28c7776c662fcb776b430f05, it was sharing a whole `cli.option(...)` like now and I don't see `--json` option.

Are you talking about something happened before the config schema?